src/omnisharp/protocol.ts

```diff
@@ -431,7 +431,7 @@ export interface UnresolvedDependenciesMessage {
 }
 
 export interface ProjectConfigurationMessage {
-    ProjectGuid: string;
+    ProjectId: string;
```

Comment:
Was this the bit causing the telemetry issues or is this just cleanup?

Reply:
This is the clean up step.